tests/smoke/support/api/http.ts

```diff
@@ -0,0 +1,52 @@
+import join from 'join-path'
+import fetch, { BodyInit, Response } from 'node-fetch'
+import { User } from '../types'
+import { config } from '../config'
+import _ from 'lodash'
+
+export const request = async ({
+  method,
+  path,
+  body,
+  user
+}: {
+  method: 'POST' | 'DELETE' | 'PUT' | 'GET'
+  path: string
+  body?: BodyInit
+  user?: User
+}): Promise<Response> => {
+  return await fetch(
+    join(
+      config.backendUrl,
+      'ocs',
+      'v2.php',
+      path + (path.includes('?') ? '&' : '?') + 'format=json'
+    ),
+    {
+      method,
+      body,
+      headers: {
+        'OCS-APIREQUEST': true as any,
+        ...(user && {
+          Authorization: 'Basic ' + Buffer.from(user.id + ':' + user.password).toString('base64')
+        })
+      }
+    }
+  )
+}
+
+export const checkResponseStatus = (response: Response, message = ''): void => {
+  // response.status >= 200 && response.status < 300
```

Comment:
Dev leftover?

Reply:
think yes, should be fine to get rid of it